package.json

```diff
@@ -41,6 +42,7 @@
         "@mdi/font": "7.0.96",
         "@nordicsemiconductor/nrf-device-lib-js": "^0.4.13",
         "@reduxjs/toolkit": "1.8.3",
+        "@svgr/core": "^6.5.1",
```

Comment:
Same as in the launcher, shall we pin them?

Reply:
Good point, will do it when I remove webpack.